dist/jquery.panzoom.js

```diff
@@ -1125,6 +1128,10 @@
 			if (this.panning) {
 				return;
 			}
+
+			var type = event.type;
+			if (window.PointerEvents&&(type === 'touchstart')) return false;  // fix double events pointer/touch on Chrome >=55 #303
```

Comment:
window.PointerEvents??? Is there such a thing? I think this is a typo. And you meant window.PointerEvent. See https://developers.google.com/web/updates/2016/10/pointer-events